src/components/Login/Login.jsx

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios';
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import { useAppContext } from '../../AppContext';
+
+import { Button } from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+
+import './Login.css';
+
+const baseUrl = 'http://localhost:3000';
+
+export default function Login() {
+	const { setUserName } = useAppContext();
+	const [userEmail, setUserEmail] = useState('');
+	const [userPassword, setUserPassword] = useState('');
```

Comment:
Please check out this video https://www.youtube.com/watch?v=4UZrsTqkcW4&t=22871s from previous topic. There's nice way to deal with multiple input fields)

Reply:
Done!